pkg/cvo/sync_worker.go

```diff
@@ -1031,7 +1022,7 @@ func runThrottledStatusNotifier(ctx context.Context, interval time.Duration, buc
 				return
 			case next := <-ch:
 				// only throttle if we aren't on an edge
-				if next.Generation == last.Generation && next.Actual.Image == last.Actual.Image && next.Reconciling == last.Reconciling && (next.Failure != nil) == (last.Failure != nil) {
+				if next.Generation == last.Generation && next.Actual.Image == last.Actual.Image && next.State == last.State && (next.Failure != nil) == (last.Failure != nil) {
```

Comment:
Since the semantics have changed are you sure that this will never result in a change in behavior? I.e. whereas before it was limited to whether the reconciling bool differed and now it's been widened to multiple possible State values, e.g. next.State == ReconcilingPayload, last.State == InitializingPayload or some such.